test/model/binding.dart

```diff
@@ -513,6 +509,11 @@ class FakeAndroidNotificationHostApi implements AndroidNotificationHostApi {
     _activeNotificationsMessagingStyle.clear();
   }
 
+  @override
+  Future<void> createNotificationChannel(NotificationChannel channel) async {
```

Comment:
nit: keep this next to the data it acts on (so `_createdChannels`)

Reply:
Relatedly, in the first commit this should go above `takeNotifyCalls`, so as not to separate that from `notify`.